lib/index.js

```diff
@@ -85,25 +95,96 @@ Vero.prototype.track = function(payload, fn) {
  * @api public
  */
 
-Vero.prototype.alias = request('/users/reidentify', 'put');
+Vero.prototype.alias = request('alias', '/users/reidentify', 'put');
+
+/**
+ * Add or Remove Tags.
+ *
+ * http://developers.getvero.com/?bash#tags
+ *
+ * @param {Facade} msg
+ * @param {Object} tags
+ * @param {Function} fn
+ * @api public
+ */
+
+Vero.prototype.addOrRemoveTags = function(msg, tags, fn) {
+  // Check to ensure tags object is properly formatted.
+  var errmsg;
+  if (Object.prototype.toString.call(tags) !== '[object Object]') {
+    errmsg = 'Tag option must be passed as an Object.';
```

Comment:
we need to set a status code to prevent the unknown errors in datadog/loggly

would probably just refactor this to something like:

```
var err;
if (...) {
   err = new Error('<msg>');
   err.status = 400;
   return fn(err);
}
```

and do that for the rest of the `if`s as well :)